how-to-guides/re-run-failed-training/scripts/re_run_failed_training.py

```diff
@@ -0,0 +1,113 @@
+import neptune.new as neptune
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import os
+from torchvision import datasets, transforms
+
+# Step 1: Get Run ID
+
+# Fetch project
```

Comment:
Here you create project object, you do not fetch it

Reply:
The definition of neptune.get_project() says

 - > "Get a project with given name." 

So that's why I wrote fetch a project, it makes it easier to understand.

I know that technically you are creating a project object that can be used to interact with the project as a whole and one example of it would be fetching data from RunsTable.

But if you say it's creating a project it will be confusing because we have a method to actually create a project 
https://docs.neptune.ai/api-reference/management#.create_project